casper/src/test/scala/coop/rchain/casper/ValidateTest.scala

```diff
@@ -792,6 +792,52 @@ class ValidateTest
       } yield result
   }
 
+  "Block validDeploySignatures" should "return Ok if all deploys are signed correctly" in withStorage {
```

Comment:
sed 's/return Ok/succeed' because no `Ok` is being returned anywhere

Reply:
👍 fixed